external/fv3fit/fv3fit/keras/_models/recurrent.py

```diff
@@ -0,0 +1,733 @@
+from typing import Dict, Any, Iterable, Hashable, Optional, Sequence, Tuple, List
```

Comment:
This is mostly ok, but I have several nits. I think we should do a pair programming session to address them.